cmd/frontend/graphqlbackend/org_invitations.go

```diff
@@ -562,6 +569,17 @@ func sendOrgInvitationNotification(ctx context.Context, db database.DB, org *typ
 	})
 }
 
+func subjectMatchesOrgId(claims *orgInvitationClaims, invite *database.OrgInvitation) error {
+	orgID, err := strconv.ParseInt(claims.Subject, 10, 32)
+	if err != nil {
+		return err
+	}
+	if orgID != int64(invite.OrgID) {
```

Comment:
since the original orgID can only be 32 bit integer, this seems like a better conversion to me:
```suggestion
	if int32(orgID) != invite.OrgID {
```